### PR TITLE
Cypress Dropdown Fix

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

Due to the addition of a deletable Adult Core Set, the cypress test have started to fail. The delete code we were using only had to worry about one core set per user as there would only be one set of deletes in the system. Because adult measures have a delete option now, if both adult and child cypress is run at the same time and is generated at the same time, it creates an issue.

We were using:  `cy.get('[data-cy="Delete"]').first().click({ force: true });` to delete the first in the stack order, but this is indifferent to whether the button is visible or not. So it didn't matter if the child menu or the adult menu was made visible, we would still delete the adult measure as that menu still exist even if it is hidden; it will usually be the first in the stack. 

Screenshot 2024-05-10 at 12 53 35 PM

Another issue is that things would be animating while trying to click reset / complete all the measure buttons, which would cause misfires to the wrong button in the menu. So this code was added to some of the buttons
```
{
      force: true,
      waitForAnimations: false,
}
```

### Related ticket(s)

N/A

---
### How to test

If the cypress test passes, we should be good.

### Notes

N/A

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: tests/cypress/support/commands.ts [hunk 1]
```diff
@@ -145,34 +145,48 @@ Cypress.Commands.add("displaysSectionsWhenUserNotReporting", () => {
   ).should("be.visible");
 });
 
-// helper recursive function to remove added core sets
-const removeCoreSetElements = (kebab: string) => {
+const clickCoreSetAction = (kebab: string, action: string) => {
```
Comment (gmrabian) [inline tests/cypress/support/commands.ts:148]: `action` is a little misleading. this is just a selector, right?

**Review by gmrabian — APPROVED** [review]
this resolves the issues I saw with the tests

File: tests/cypress/support/commands.ts (hunk 1 above)
Comment (ailZhou) [reply tests/cypress/support/commands.ts:148]: true, but we call that column core set actions 🤔

Comment (ailZhou) [reply tests/cypress/support/commands.ts:148]: i am willing to rename this, but in a different PR because i don't want to wait for this to compile again

**Review by BearHanded — APPROVED** [review]

**Pull request merged**